src/main/resources/api/schemas/calendar.yaml

```diff
@@ -0,0 +1,71 @@
+description: A single calendar
```

Comment:
There is a convention in FOLIO that records have some [change metadata|https://github.com/folio-org/raml/blob/raml1.0/schemas/metadata.schema] about when and which user created / changed a record.

I would think that folks would want the records in this module to follow that convention.

Reply:
I had not seen any records in the previous mod-calendar with this -- do you have any links to docs about FOLIO standards on this?  I've heard some conflicting information (especially as far as privacy concerns go) and want to make sure I implement this properly

Reply:
> I had not seen any records in the previous mod-calendar with this

Probably because either the convention came about afterwards or the developers of mod-calendar were unaware of it.

> do you have any links to docs about FOLIO standards on this?

The [link to the RAML standards repository|https://github.com/folio-org/raml/blob/raml1.0/schemas/metadata.schema] above describes the current standard.

Admittedly, FOLIO does not have a good, single place for documentation about module expectations and so piecing all of this together is a royal faff.

> I've heard some conflicting information (especially as far as privacy concerns go) and want to make sure I implement this properly

Who have you heard these concerns from?

It's pretty much a standard request from POs nowadays I believe (and why pretty much every module stores some personal information).

Reply:
> Probably because either the convention came about afterwards or the developers of mod-calendar were unaware of it.

I saw that mod-calendar used an auditing feature from RMB, but that [no other modules](https://github.com/search?q=org%3Afolio-org+withauditing&type=Code) used it (although, looking back, it looks like there is a new `withMetadata` feature that succeeded it?). 

I don't remember who we met with when we discussed this, but we decided that it was not a big enough need to be included in the initial featureset.  The number of times calendars would be edited, as well as the number of users who should have edit/create access, would be quite small, so we didn't identify it as an immediate need.  We didn't realize that there was a FOLIO standard for this kind of information; I'll work on updating the code to capture this data.

Reply:
I've added this feature; since it touches on a few places you have already reviewed, here's the single commit: https://github.com/folio-org/mod-calendar/commit/28e9d605ecaab7d922e33e971d54117a7a697a67

Reply:
> I saw that mod-calendar used an auditing feature from RMB

I believe auditing feature in RMB does something different, I think it makes copies of the record to a separate table on each change.

> We didn't realize that there was a FOLIO standard for this kind of information; I'll work on updating the code to capture this data.

Thanks. It is hard, there isn't a single place to learn about these kinds of conventions / standards.